app/views/pay/user_mailer/payment_failed.html.erb

```diff
@@ -0,0 +1,7 @@
+<h3>Processing your payment failed</h3>
+<p>
+  Your <%= Pay.business_name %> subscription payment failed. 
+  Please update your payment information.
+</p>
+
+<p>- The <%= Pay.business_name %> Team</p>
```

Comment:
Some inspiration for content. This is one I send out currently, and they get more urgent after a couple attempts.

> Hey Collin,
> 
> Chris from GoRails here. I am writing because it looks like your subscription payment of $19 didn't go through. > Please [update your information here](https://gorails.com) and we'll give it another try.
> 
> Let us know if you have any questions,
> 
> —
> Chris
> Founder, GoRails

We can tweak this and include a link to the `root_url` for the user to click. We could try to make it feel friendly and use "The [business-name] team" or we could add another config for a name and include that (if present) to make it more human.

Reply:
Thanks for the thoughts here, I appreciate it. I was going to get with you about writing a nice email for this. This was just my placeholder/starting point for the beginning. :)

I'll cook up a updated one inspired by the comments here, and perhaps look at other dunning email examples elsewhere.